CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ CHANGELOG
 - Track the longest dynamic node idle time in CloudWatch Dashboard.
 - Add new configuration section `HealthChecks/Gpu` for enabling the GPU Health Check in the compute node before job submission.
 - Add support for `Tags` in the `SlurmQueues` section.
+- Add support for the detailed monitoring configuration for Slurm compute fleet.
```

Comment:
What about: Add support for `EnableDetailedMonitoring` in the `Monitoring` section.

Reply:
sure. updated.